fission-web/Main.hs

```diff
@@ -38,13 +40,19 @@ main = do
   let
     Storage.Environment {..} = env ^. storage
     Web.Environment     {..} = env ^. web
+    AWS.Environment     {..} = env ^. aws
 
     _herokuID       = Hku.ID       . encodeUtf8 $ manifest ^. Hku.id
     _herokuPassword = Hku.Password . encodeUtf8 $ manifest ^. Hku.api ^. Hku.password
 
     _ipfsPath    = env ^. ipfs . binPath
     _ipfsURL     = env ^. ipfs . url
     _ipfsTimeout = env ^. ipfs . IPFS.timeout
+    
+    _awsAccessKey = env ^. aws . AWS.accessKey
+    _awsSecretKey = env ^. aws . AWS.secretKey
+    _awsZoneId    = env ^. aws . AWS.zoneId
+    _awsDomain    = env ^. aws . AWS.domain
```

Comment:
We're brought all of these into scope already above...

```haskell
AWS.Environment     {..} = env ^. aws
```

...so we don't need to do all of this lensing.

## Option 1

Remove the above `AWS.Environment {..}` line

## Option 2

```haskell
_awsAccessKey = _accessKey
_awsSecretKey = _secretKey
_awsZoneId    = _zoneId
_awsDomain    = _domain
```